docs/api/FluxibleContext.md

```diff
@@ -19,7 +42,7 @@ Component({
 });
 ```
 
-## executeAction(action, payload, callback)
+### executeAction(action, payload, callback)
 
 This is the entry point into an application's execution. The initial action is what begins the flux flow: action dispatches events to stores and stores update their data structures. On the server, we wait for the initial action to finish and then we're ready to render using React. On the client, the components are already rendered and are waiting for store change events.
 
```

Comment:
The extra space after the bullets (outside of this commit) is causing the code snippet to be indented with the last bullet.